Bottom: c298c9e5ce48c7cf9ec0f2933b49f4c604861f9e
Top:    d45fbf12fa4999cfec0baedf939612f916f3bba6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 00:32:42 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
